app/assets/stylesheets/application.css.scss.erb

```diff
@@ -318,3 +318,9 @@ $service-colors: (
   color: yellow;
   text-decoration: underline;
 }
+
+.select2-results .fa {
+  float: right;
```

Comment:
It looks good already, would it make sense to move the icon to the left of the text?